### DISCUSSION TITLE
build error - what am I doing wrong?

### DISCUSSION THREAD

**Discussion by krv76** [boostorg/geometry#1203] [Q&A]
Hi! Could you help me please?

After include <boost/geometry/extensions/index/rtree/rtree.hp> I got the following:
...
In file included from /home/rkukushkin/projects/3d-core/tests/src/geometric_primitive/gp_intersection_test.cpp:21:
/home/rkukushkin/geometry/include/boost/geometry/extensions/index/rtree/rtree.hpp:32:1: error: too few template parameters in template redeclaration
template <typename Box, typename Value >
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/rkukushkin/geometry/include/boost/geometry/index/rtree.hpp:164:1: note: previous template declaration is here
template
^~~~~~~~
...
What am I doing wrong?

Thank you!

**Comment by awulkiew** [comment]
Do you really want to use the first implementation of the rtree from the extensions done during GSoC?
If the answer is "no" or "I don't know" then don't include `<boost/geometry/extensions/index/rtree/rtree.hpp>`
`<boost/geometry.hpp>` should be enough
